packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -0,0 +1,102 @@
+import type { ThemeTokens } from '../../../tokens/theme/theme.d';
+import type { TypographyPlatforms } from '../../../tokens/global/typography';
+import type { Required } from '../../../_helpers/types';
+import type { IconProps } from '../../Icons';
+import type { BaseButtonProps } from './BaseButton';
+
+export type ButtonMinHeight = 48 | 40 | 32 | 28;
+
+export type ButtonTypography = {
+  [key in TypographyPlatforms]: {
+    fonts: {
+      size: Record<
+        Required<BaseButtonProps['size']>,
+        keyof ThemeTokens['typography'][key]['fonts']['size']
+      >;
+    };
+    lineHeights: Record<
+      Required<BaseButtonProps['size']>,
+      keyof ThemeTokens['typography'][key]['lineHeights']
+    >;
+  };
+};
+
+const typography: ButtonTypography = {
+  onDesktop: {
+    fonts: {
+      size: {
+        large: 200,
+        medium: 100,
+        small: 75,
+        xsmall: 75,
+      },
+    },
+    lineHeights: {
+      large: 's',
+      medium: 'l',
+      small: 'l',
+      xsmall: 'l',
```

Comment:
nit(sorry but not sorry): sort the value? xsmall ➡️ small ➡️ medium and so on for all the objects in this file. 🙈

Reply:
Will do this later on, I'll add a comment in the file to remember. Its not just here but in other places too where I need to sort things by name. Avoiding such things since it causes a huge mess in rebasing chained PRs.